TriangulationForm.html

```diff
@@ -466,6 +500,56 @@ <h3>SECÇÃO DE TRIANGULAÇÃO</h3>
 		}
 	});
 }
+
+var gaac = htmlDataStore.getObsByConceptId(patientuuid,23724);  
+gaac = JSON.parse(gaac);
+var gaacSize = gaac.length-1;
+if(parseInt(gaacSize)>-1){
+    $("#gaac").val(gaac[0].valueCoded.name);
+}
+
+var familyApproach = htmlDataStore.getObsByConceptId(patientuuid,23725);  
+familyApproach = JSON.parse(familyApproach);
+var familyApproachSize = familyApproach.length-1;
+if(parseInt(familyApproachSize)>-1){
+    $("#familyApproach").val(familyApproach[0].valueCoded.name);
+}
+
+var tb = htmlDataStore.getObsByConceptId(patientuuid,58);  
+tb = JSON.parse(tb);
+var tbSize = tb.length-1;
+if(parseInt(tbSize)>-1){
+    $("#tb").val(tb[0].valueCoded.name);
+}
+
+var lastVisit = htmlDataStore.getObsByConceptId(patientuuid,23724);  
+lastVisit = JSON.parse(lastVisit);
+var lastVisitSize = lastVisit.length-1;
+if(parseInt(tbSize)>-1){
+    $("#lastVisit").val(lastVisit[0].valueCoded.name);
+}
+
+var nextVisit = htmlDataStore.getObsByConceptId(patientuuid,1410);  
```

Comment:
this is datetime, use valueDateTime, refer to this https://github.com/muzima/muzima-android/blob/master/app/src/main/java/com/muzima/view/forms/HTMLFormDataStore.java#L661-L724